apps/server/src/main.js

```diff
@@ -5,6 +5,16 @@
 
 import express from 'express';
 import * as path from 'path';
+const router = require('express').Router();
+const apiRoutes = require('./api');
+const userRoutes = require('./userRoutes');
+
+router.use('/api', apiRoutes);
+router.use('/users', userRoutes);
```

Comment:
@AMess33 I think you want your user routes to be inside your api routes as well. That way you know any route prefixed with `/api` is a server route and any other request should just return your React app so the route can be handled by react-router.